templates/frontend/pages/indexJournal.tpl

```diff
@@ -21,9 +21,21 @@
 {include file="frontend/components/header.tpl" pageTitleTranslated=$currentJournal->getLocalizedName()}
 
 <main class="page_index_journal">
-	{* Display homepage image if set *}
+
+	{* Display homepage image if set, and wrap around journal summary if use chooses to display it *}
+	{if $homepageImage}
+		<div class="homepage_image" style="background-image: url('{$publicFilesDir}/{$homepageImage.uploadName|escape:"url"}')">
+	{/if}
+
+	{if $showJournalSummary}
```

Comment:
Here and in other occurrences, I would also check if there is an actual input for the journal description, e.g., there can be a situation when setting is `on` but the description is empty.

Reply:
See commit https://github.com/pkp/classic/commit/6a1b3842301f06db15c94d3495916d786067a59e and the following conversation: https://github.com/pkp/classic/pull/37#pullrequestreview-276546486